stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewType.kt

```diff
@@ -19,4 +19,5 @@ public object MessageListItemViewType {
     public const val ERROR_MESSAGE: Int = OFFSET + 10
     public const val THREAD_PLACEHOLDER: Int = OFFSET + 11
     public const val GIPHY_ATTACHMENT: Int = OFFSET + 12
+    public const val IMAGE_ATTACHMENT: Int = OFFSET + 13
```

Comment:
we'll have to double check these once the link attachments are merged

Reply:
@bychkovdmitry can keep 13. 
I'll take 14 and 15.